R/read_package.R

```diff
@@ -64,5 +61,5 @@ read_package <- function(file = "datapackage.json") {
   }
   cli::cli_inform(message, class = "frictionless_message_usage_rights")
 
-  descriptor
+  create_package(descriptor)
```

Comment:
Why not:
```suggestion
return(create_package(descriptor))
```

Like for `create_package()`?

Reply:
I mainly want to avoid "naked" variables. So I don't see a need to double wrap here.